### PR TITLE
MCG namespace tests - tier4

### PR DESCRIPTION
**Pull request by fbalak**

### CODE REVIEW

File: tests/manage/mcg/test_namespace.py [hunk 1]
```diff
@@ -103,7 +105,127 @@ def test_write_to_ns_read_from_aws(self, mcg_obj, cld_mgr, awscli_pod, ns_resour
         self.download_files(mcg_obj, awscli_pod, bucket_to_read=target_bucket_name, s3_creds=s3_creds)
 
         # Compare between uploaded files and downloaded files
-        self.compare_dirs(awscli_pod, amount=3)
+        assert self.compare_dirs(awscli_pod, amount=3)
+
+    @tier4
```
Comment (ebenahar) [inline tests/manage/mcg/test_namespace.py:110]: missing tier4a/b/c

File: tests/manage/mcg/test_namespace.py [hunk 2]
```diff
@@ -103,7 +105,127 @@ def test_write_to_ns_read_from_aws(self, mcg_obj, cld_mgr, awscli_pod, ns_resour
...
+        )
+
+        # Read files directly from AWS
+        self.download_files(mcg_obj, awscli_pod, bucket_to_read=target_bucket_name, s3_creds=s3_creds)
+
+        # Compare between uploaded files and downloaded files
+        assert self.compare_dirs(awscli_pod, amount=3)
+
+    @pytest.mark.polarion_id("OCS-2293")
+    @tier4
```
Comment (ebenahar) [inline tests/manage/mcg/test_namespace.py:163]: same

**Comment by ebenahar** [comment]
@nimrod-becker @romayalon

**Review by parikshithb — DISMISSED** [review]
Looks great

**Review by ebenahar — APPROVED** [review]

**Comment by ebenahar** [comment]
@fbalak verified?

**Review by ebenahar — APPROVED** [review]

**Review by parikshithb — APPROVED** [review]

**Pull request merged**